sessions/recap-project-5/example-implementation/components/CommentForm.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+`;
+
+const Button = styled.button`
+  appearance: none;
+  border: none;
+  background: #000;
+  color: #fff;
+  padding: 0.5rem 1rem 0.5rem 1rem;
+  width: fit-content;
+`;
+
+const TextArea = styled.textarea`
+  width: 70%;
+  padding: 1rem;
+  border-radius: 20px 20px 20px 0;
+`;
```

Comment:
we should prevent resizing in both directions
```suggestion
  resize: vertical;
`;
```